src/modules/obj.js

```diff
@@ -0,0 +1,7 @@
+export default class Obj {
+  constructor(NewValue, index) {
+    this.inputValue = NewValue;
+    this.indexNumber = index;
+    this.completed = true;
+  }
```

Comment:
I sugest keepin the task completed false  by default. just because it less confusing to add a completed task to a todo list.